tests/app/questionnaire/test_router.py

```diff
@@ -793,6 +793,7 @@ def test_return_to_grand_calculated_summary_from_answer(
             return_to="grand-calculated-summary",
             block_id="distance-calculated-summary-1",
             return_to_block_id="distance-grand-calculated-summary",
+            return_to_answer_id="distance-calculated-summary-1,distance-calculated-summary-1",
             _anchor="distance-calculated-summary-1",
```

Comment:
same comment again, this test looks off

Reply:
This issue disappears when I implement your code suggestion, so resolved.